Remove download option for konect instances

duylethanh opened the pull request:
Resolves #85.

One thing that we should maybe reconsider is the removal of the konect case in ``convert_to_edgelist()`` located in the ``instances.py``. Users might have already downloaded konect instances. 

Alternatively, we could only remove the option of downloading konect instances and leave the rest. In this case users can still use ``simex instances run-transform`` manually on the instances.

avdgrinten commented:
I think in the long term, we should move the graph-specific code into some kind of plugin system since Simexpal should not only focus on graphs but on arbitrary algorithmic experiments.

For now the second option (only disable downloading, keep the remaining logic) sounds reasonable.

duylethanh commented:
PR updated. I removed the download option for konect instances and adjusted the documentation accordingly. I also added the ``raise`` keyword to the ``DownloadException`` as it was just hanging around.

avdgrinten reviewed (approved):
LGTM.
